install/set-up-database.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+source "$(dirname $0)/_lib.sh"
+
+echo "${_group}Setting up database ..."
```

Comment:
We should probably say `Setting up / migrating` here and in the file name.

Reply:
Done in 676e3b2ddc04bf6af65d3339f1924630e4e7dbdc.